cypress-tests/cypress/support/commands.js

```diff
@@ -3331,3 +3331,95 @@ Cypress.Commands.add("incrementalAuth", (globalState, data) => {
     }
   });
 });
+
+Cypress.Commands.add("createConfigs", (globalState, key, value) => {
+  const base_url = globalState.get("baseUrl");
+  const api_key = globalState.get("adminApiKey");
+
+  cy.request({
+    method: "POST",
+    url: `${base_url}/configs/`,
+    headers: {
+      "Content-Type": "application/json",
+      "api-key": api_key,
+    },
+    body: {
+      key: key,
+      value: value,
+    },
+    failOnStatusCode: false,
+  }).then((response) => {
+    logRequestId(response.headers["x-request-id"]);
+
+    expect(response.status).to.equal(200);
+    expect(response.body).to.have.property("key").to.equal(key);
+    expect(response.body).to.have.property("value").to.equal(value);
+  });
+});
```

Comment:
when creating configs, we're not passing the key as a param in the `url` but as a kv pair in the body.

i feel, we can update the existing `updateConfig` function to support creating configs as well (by passing another value to the function, something like `create`, `update`, `delete` and based on the check, either create a new config or update the existing one)? we can also pass relevant `method` based on this check.

but yeah, this check has to be done the beginning of the function, even before declaring the constants and variables.

Reply:
In my opinion, this test case is rather simple and even though there may be some code duplication, it's minimal. I think we need not consider optimizing it now.